### ISSUE TITLE
A way to export/import organization settings

### ISSUE THREAD

**Issue by rvmourik** [p2-inc/keycloak-orgs#50]
Hi,

I might be wrong, but if I were to export the realm/client settings with keycloak cli it doesn't export information regarding the organization that are present, right?

Reason why I am asking is that for our development environment I would like to create an import file which imports realm/client/organization for development purposes on starting the docker containers. This way every developer has the same settings regarding organizations.

Is this something that is possible?

Regards,

Robbert

**Comment by xgp** [comment]
That's correct. We weren't able to figure out a way to hook into the realm import/export functionality.

Right now, our recommendation is to back up all of the Organization tables in the database and use the dump file to restore. Because we use realm name as our foreign key, there shouldn't be a problem if the UUIDs in the Keycloak tables are different.
